test/integration_test.py

```diff
@@ -742,14 +759,38 @@ def _list_sources(http_client: RequestMethods):
                        sourceId=tdr_client.lookup_source_id(source_spec))
             for source_spec in map(TDRSourceSpec.parse, config.tdr_sources(catalog))
         }
-        self.assertEqual(_list_sources(tdr_client._http_client), all_sources)
+        unauthenticated_sources = _list_sources()
```

Comment:
Nit: It's not the sources that are "unauthenticated", but the user, right? A name like `public_access_sources` matches `managed_access_sources` and I think makes more sense.